[korean_rr] remove non-functional Windows target

DavidLRowe opened the pull request.

DavidLRowe commented:
Addresses the immediate issue raised on #1703 by removing Windows as a target.

Does not address Keyman's lack of IMX support.

mcdurdin commented:
> Does not address Keyman's lack of IMX support.

Keyman for Windows has IMX support. So I think the big issue here is that I haven't had space to look into this problem, because there's a bunch of history here around this keyboard, and I don't remember it. I understand the Windows target is not working but would like to try and fix that rather than simply disable it and then forget again :)

DavidLRowe commented:
>    Does not address Keyman's lack of IMX support.

> Keyman for Windows has IMX support. So I think the big issue here is that I haven't had space to look into this problem, because there's a bunch of history here around this keyboard, and I don't remember it. I understand the Windows target is not working but would like to try and fix that rather than simply disable it and then forget again :)

Sorry for the sloppy comment. Perhaps "Does not address this keyboard's IMX issues on Windows."? (Instead this PR removes Windows as a target.)

Issue #1703 will remain open.

DavidLRowe commented:
@rc-swag 
When I changed the target to web only, the .kmx file wasn't generated, which caused an error.
I removed the .kmx from the .kps list, and got an error saying it needed either .kmx or .js.
I added the .js and it built.
But the question is whether this package has everything needed to work on the web. Is it possible to build a web-only keyboard?

LornaSIL commented:
@darcywong00 I don't have enough knowledge to know if this should be approved or not.

darcywong00 commented on release/k/korean_rr/source/korean_rr.kps:
```diff
@@ -9,7 +9,6 @@
     <ReadMeFile>readme.htm</ReadMeFile>
     <MSIFileName></MSIFileName>
     <MSIOptions></MSIOptions>
-    <FollowKeyboardVersion/>
```
@DavidLRowe - is it possible to keep this checked?

rc-swag replied:
@LornaSIL  @DavidLRowe  I think we need to see if we can keep this checked then I am ok to approve

darcywong00 commented:
> @darcywong00 I don't have enough knowledge to know if this should be approved or not.

I think this one is fine to approve. @DavidLRowe is keeping #1703 open and I also created a related issue keymanapp/keyman#9516 for refactoring Keyman for Windows to support korean_rr.

LornaSIL commented:
It won't let me check it, so I think @rc-swag needs to approve it now.

rc-swag reviewed (approved):
lgtm
